ArduPlane/quadplane.cpp

```diff
@@ -1258,7 +1268,7 @@ void QuadPlane::control_loiter()
     if (throttle_wait) {
         motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::GROUND_IDLE);
         attitude_control->set_throttle_out(0, true, 0);
-        attitude_control->relax_attitude_controllers();
+        relax_attitude_control();
```

Comment:
There are lots more calls to `attitude_control->relax_attitude_controllers();` You could replace them all I think and remove the tailsitter checks on some of them as its now done in the new function.

Reply:
But we only want the new behavior when throttle_wait is true, right?
I must be missing your point.

Reply:
The other calls already have the !tailsitter check, you can remove that check and just use the new function. We then benefit from relaxing roll and pitch in all cases.

Reply:
do you mean never relaxing pitch for vectored tailsitters?

Reply:
Sorry, yeah, typo, we benefit from relaxing roll and YAW. Basically I think all calls to `attitude_control->relax_attitude_controllers();` can be replaced by your new function. 

This for example, https://github.com/ArduPilot/ardupilot/blob/56e44c46ca3202b8fee52cebff85faf3a2c0d12c/ArduPlane/quadplane.cpp#L925

There may be others too.

Reply:
I see now, there are places where roll and yaw are not being relaxed when they probably should be, and this would be why we see max elevon deflection before takeoff in qstabilize.

And I see now that throttle_wait is completely ignored by hold_stabilize, which seems to make setting it to false in init_stabilize a red herring.